recipes/tiberius/build.sh

```diff
@@ -4,4 +4,12 @@ mkdir -p ${PREFIX}/bin
 cp -R $SRC_DIR/bin/* ${PREFIX}/bin/
 chmod +x ${PREFIX}/bin/tiberius.py
 ln -s ${PREFIX}/bin/tiberius.py ${PREFIX}/bin/tiberius
-sed -i '1s|^|#!/usr/bin/env python3\n|' ${PREFIX}/bin/tiberius
\ No newline at end of file
+sed -i '1s|^|#!/usr/bin/env python3\n|' ${PREFIX}/bin/tiberius
+
+#Temporary: until the specific branch of learnMSA compatible with Tiberius is merged
+#Adding learnMSA
+git clone https://github.com/Gaius-Augustus/learnMSA
```

Comment:
You could use a URL in the meta.yml like this: https://github.com/Gaius-Augustus/learnMSA/archive/refs/heads/parallel.tar.gz

However, we should not release branches that are not in main IMHO. maybe a new package?

Reply:
On the tiberius repository, they indicate to use the parallel branch to run the tool. I asked the question on leanrMSA's repository. Here's their answer: https://github.com/Gaius-Augustus/learnMSA/issues/12#issuecomment-2386247553